### ISSUE TITLE
Extract RAR file error

### ISSUE THREAD

**Issue by DrobnjakS18** [wapmorgan/UnifiedArchive#31]
I have successfuly extract RAR file with  `extractFiles()` but the error is still been thrown.

Error:
"Could not extract archive: The command \"'/usr/bin/7z' 'x' '/var/www/project/storage/app/tmp/rUAl2hT4NZjkbqQIMHvmutUNPGNIOhTgzpZAOdNL.rar' '-aoa' '-o/var/www/project/storage/app/tmp/test' '-y' '-p '\" failed.\n\nExit Code: 2(Misuse of shell builtins)\n\nWorking directory: /var/www/project/public\n\nOutput:\n================\n\n7-Zip [64] 16.02 : Copyright (c) 1999-2016 Igor Pavlov : 2016-05-21\np7zip Version 16.02 (locale=C,Utf16=off,HugeFiles=on,64 bits,1 CPU AMD FX(tm)-6100 Six-Core Processor              (600F12),ASM,AES-NI)\n\nScanning the drive for archives:\n1 file, 4359357 bytes (4258 KiB)\n\nExtracting archive: /var/www/project/storage/app/tmp/rUAl2hT4NZjkbqQIMHvmutUNPGNIOhTgzpZAOdNL.rar\n--\nPath = /var/www/project/storage/app/tmp/rUAl2hT4NZjkbqQIMHvmutUNPGNIOhTgzpZAOdNL.rar\nType = Rar5\nPhysical Size = 4359357\nSolid = -\nBlocks = 7\nEncrypted = -\nMultivolume = -\nVolumes = 1\n\n\nSub items Errors: 3\n\nArchives with Errors: 1\n\nSub items Errors: 3\n\n\nError Output:\n================\nERROR: Unsupported Method : jsRar/images_1/testCopy.png\nERROR: Unsupported Method : jsRar/images_1/testCopy2.jpg\nERROR: Unsupported Method : jsRar/images_2/eBa.png\n"

**Comment by wapmorgan** [comment]
It tells
```Sub items Errors: 3

Archives with Errors: 1

Sub items Errors: 3


Error Output:
================
ERROR: Unsupported Method : jsRar/images_1/testCopy.png
ERROR: Unsupported Method : jsRar/images_1/testCopy2.jpg
ERROR: Unsupported Method : jsRar/images_2/eBa.png```

I think that means that the compression method used for these files is not supported by 7z.
And thus can't be fixed by UnifiedArchive

**Comment by wapmorgan** [comment]
As I can see, SevenZip can't handle rar's.
Fixed It in 1.1.4.

**Comment by wapmorgan** [comment]
@DrobnjakS18 figured out that sevenZip can work with rar, you need to install 7zip codecs.
In Ubuntu, its package **p7zip-rar**. Eventually, you should install 7z utility that can open rar.
